packages/kbn-unified-data-table/src/components/__snapshots__/data_table_columns.test.tsx.snap

```diff
@@ -468,7 +319,7 @@ Array [
     "displayAsText": "message",
     "id": "message",
     "isSortable": true,
-    "schema": "string",
+    "schema": "kibana-json",
```

Comment:
It seems to be incorrect that schema got changed in the tests. What did trigger it?

Reply:
You were not applying columnsMeta in the test, this is why it has this value. I updated the test to give the correct configuration so now the snapshot will be correct.

Reply:
https://github.com/elastic/kibana/pull/181399/commits/a6ddb094a5490fe1f2255d6cb4961e0817ace9f5